packages/edgeidentity/ios/src/RCTAEPEdgeIdentityDataBridge.m

```diff
@@ -0,0 +1,104 @@
+ */
+
+#import "RCTAEPEdgeIdentityDataBridge.h"
+
+@implementation RCTAEPEdgeIdentityDataBridge
+
+static NSString* const ID_KEY = @"id";
+static NSString* const IS_PRIMARY_KEY = @"primary";
+static NSString* const AEP_AUTH_STATE_KEY = @"authenticatedState";
+static NSString* const ITEM_KEY = @"items";
+static NSString* const AUTHENTICATED = @"authenticated";
+static NSString* const LOGGED_OUT = @"loggedOut";
+static NSString* const AMBIGUOUS = @"ambiguous";
+
++ (NSDictionary *)dictionaryFromIdentityMap: (nullable AEPIdentityMap *) idmap {
+    NSMutableDictionary *mapDict = [NSMutableDictionary dictionary];
+    
+        for (NSString *namespace in idmap.namespaces) {
+            NSArray* items = [idmap getItemsWithNamespace:namespace];
+           
+            NSMutableArray *mapArray = [NSMutableArray array];
+            
+            for (AEPIdentityItem *item in items){
+              NSMutableDictionary *itemdict = [NSMutableDictionary dictionary];
+              itemdict[IS_PRIMARY_KEY] = @(item.primary);
+              itemdict[AEP_AUTH_STATE_KEY] = stringFromAuthState(item.authenticatedState);
+              itemdict[ID_KEY] = item.id ;
+                
+              [mapArray addObject:itemdict];
+            }
+            if (mapArray.count !=0) {
+                [mapDict setObject:mapArray forKey:namespace];
+            }
+        }
+      
+        return mapDict;
+    }
+
++ (AEPIdentityMap *)dictionaryToIdentityMap: (nonnull NSDictionary *) dict {
+    NSDictionary *itemsMap = [[dict objectForKey:ITEM_KEY] isKindOfClass:[NSDictionary class]] ? [dict objectForKey:ITEM_KEY] : nil;
+    NSArray <NSString*>* namespaces = [itemsMap allKeys];
+    AEPIdentityMap *identityMap = [[AEPIdentityMap alloc] init];
+    for (NSString *namespace in namespaces){
+        NSArray* items = [itemsMap objectForKey:namespace];
+        for (NSDictionary *itemMap in items){
+            AEPIdentityItem *item = [RCTAEPEdgeIdentityDataBridge dictionaryToIdentityItem:itemMap];
+            
+            if (item){
+                [identityMap addItem:item withNamespace:namespace];
+            }
+        }
+    }
+    
+    return identityMap;
+  }
+
++ (AEPIdentityItem *)dictionaryToIdentityItem: (nonnull NSDictionary *) dict {
+    
+    NSString *identifier = [[dict objectForKey:ID_KEY] isKindOfClass:[NSString class]] ? [dict objectForKey:ID_KEY] : nil;
+    
+    if (!identifier){
+        return nil;
+    }
+    
+    NSString *authenticatedString = [[dict objectForKey:AEP_AUTH_STATE_KEY] isKindOfClass:[NSString class]] ? [dict objectForKey:AEP_AUTH_STATE_KEY] : nil;
+    
+    AEPAuthenticatedState authenticatedState = authStateFromString (authenticatedString);
+
+    BOOL primary = [[dict objectForKey:IS_PRIMARY_KEY] boolValue];
+
+    return [[AEPIdentityItem alloc] initWithId:identifier authenticatedState:authenticatedState primary:primary];
+  }
+
+static AEPAuthenticatedState authStateFromString(NSString* authStateString) {
```

Comment:
We should use the `+ returnType method(params)` notation for these two class (static) APIs also.